### DISCUSSION TITLE
Iran routing

### DISCUSSION THREAD

**Discussion by Daariush** [immortalwrt/homeproxy#197] [Q&A]
Hi, Since this app is made for China, is there anyway I can upload custom geoip/geosite files and make routing for Iran?
And is there a way to use json config?

**Comment by 1715173329** [comment]
Use custom routing.

**Reply by Daariush** [reply]
Where can i upload geosite or geoip for IR?

**Reply by xy97** [reply]
You can find your geosite and geoip [here](https://github.com/MetaCubeX/meta-rules-dat/tree/sing/geo), then add them to rule set.
